tests/test_cmdline.py

```diff
@@ -238,12 +224,12 @@ def test_list_imports__from_missing_file__fails_with_exit_code_2(tmp_path):
 
 
 def test_list_imports__from_empty_dir__logs_but_extracts_nothing(tmp_path):
-    # Enable log level INFO with -v
+    # Enable log level INFO with --verbose
     expect_logs = [
         f"INFO:fawltydeps.extract_imports:Finding Python files under {tmp_path}",
     ]
     output, errors, returncode = run_fawltydeps_subprocess(
-        "--list-imports", f"--code={tmp_path}", "--detailed", "-v"
+        "--list-imports", f"--code={tmp_path}", "--detailed", "--verbose"
```

Comment:
The shorthand `-v` is really nice on the command-line, but for anything that is committed to version control (i.e. anything that will be read many more times than it is written) I prefer the full name, so that there is less room for confusion.